splitio/storage/pluggable.py

```diff
@@ -70,8 +79,7 @@ def remove(self, split_name):
             return False
 
         self._pluggable_adapter.delete(self._prefix + split_name)
-        self._pluggable_adapter.decrement(self._traffic_type_prefix + split.traffic_type_name, 1)
-        if self._pluggable_adapter.get(self._traffic_type_prefix + split.traffic_type_name) == 0:
+        if self._decrease_traffic_type_count(split.traffic_type_name) == 0:
             self._pluggable_adapter.delete(self._traffic_type_prefix + split.traffic_type_name)
```

Comment:
No need to delete here, since `_decrease_traffic_type_count` is already deleting it if it gets to 0.

Reply:
will fix